docs/apis/plugintypes/blocks/index.mdx

```diff
@@ -0,0 +1,424 @@
+---
```

Comment:
I'm thinking that this should probably be an `.md` file so we can get the linting etc.

Reply:
Oh, I thought md files could not use imports nor extra tags like DbAccessPHP, Lang...

What is the difference between md and mdx then?

Reply:
I move it to md file.